### PR TITLE
[BUD-297] Feature - DB task templates

### PR DESCRIPTION
**Pull request by Tonik1204**
# [[BUD-297] Feature / DB templates ](https://digitalrig.atlassian.net/browse/BUD-XX)

## Description

- Add task templates support for ui, schema, server & db;
- Update client folder structure(components, atoms, pages);
- Add LoadingContext & DropDown components;
- Add skeletons for Task & Contact Details pages;

## Checklist

- [X] Add unit tests

## Screenshots

ezgif com-video-to-gif

### CODE REVIEW

File: packages/client/src/components/AddTaskOptions/AddTaskOptions.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import { useLoading } from 'contexts/LoadingContext';
+import { UserRole, Mutation, TaskTemplates } from '@buddy-app/schema';
+import AddTemplateIcon from '@material-ui/icons/LibraryBooks';
+import AddTaskIcon from '@material-ui/icons/LibraryAdd';
+import PlusButton from 'atoms/PlusButton';
+import DropDown, { ShowOptions, HideOptions } from 'components/DropDown';
+import { ADD_FROM_TEMPLATE } from 'graphql/add-from-template.graphql';
+import { AddTaskOptionsProps } from './types';
+import DICTIONARY from './dictionary';
+/* istanbul ignore file */
```
Comment (duiliopastorelli) [inline packages/client/src/components/AddTaskOptions/AddTaskOptions.tsx]: Is this correct?

File: packages/services/.gitignore [hunk 2]
```diff
@@ -8,5 +8,3 @@ jspm_packages
 
 # Webpack directories
 .webpack
-
-.env
```
Comment (duiliopastorelli) [inline packages/services/.gitignore]: Would this not be an extra safety measure?

File: packages/client/package.json [hunk 3]
```diff
@@ -104,16 +104,14 @@
...
     "collectCoverageFrom": [
-      "src/**/*.{tsx,ts}",
-      "!src/styles/*",
-      "!src/__mocks__/*",
-      "!src/shared/*",
-      "!src/graphql/*",
-      "!src/context/*",
-      "!src/index.tsx",
-      "!src/utils/apollo-client/",
-      "!src/serviceWorker.ts",
```
[the remark this answers is not in the document]
Comment (Tonik1204) [reply packages/client/package.json]: @duiliopastorelli don't worry about `serviceWorker.ts`, we'r not gonna include it :) 
New rules exclude any of `./src` level scripts ;)

File: packages/services/.gitignore (hunk 2 above)
Comment (Tonik1204) [reply packages/services/.gitignore]: No, that's fine - we already have `.env*` rule in this file - it's doing all the job

**Pull request merged**